app/app.go

```diff
@@ -314,7 +327,8 @@ func NewOsmosisApp(
 			app.IBCKeeper,
 		),
 	)
-	app.SetPostHandler(NewPostHandler(app.ProtoRevKeeper))
+	// UNFORKINGTODO OQ: Figure out set post handler
+	//app.SetPostHandler(NewPostHandler(app.ProtoRevKeeper))
```

Comment:
Hrmm, don't see the issue here: https://github.com/cosmos/cosmos-sdk/blob/50f936f60e51f62fb34005d50c2c70efc94484b7/baseapp/options.go#L239

Reply:
The problem is NewPostHandler returns an object of type sdk.AnteHandler, when we need an object of time postHander